### DISCUSSION TITLE
PSX games don't work

### DISCUSSION THREAD

**Discussion by mikhaelcool7** [OnionUI/Onion#1638] [Support]
### Checklist

- [X] I am reporting a bug in the Onion OS - Any issues with the closed source frontend ([MainUI](https://github.com/OnionUI/MainUI-issues)), or [RetroArch](https://github.com/libretro/RetroArch) or any of the cores should be directed to their respective repositories
- [X] I have reviewed the [Onion Docs](https://onionui.github.io/docs) and did not find the answer
- [X] I have searched the existing (open and closed) issues to ensure I do not log a duplicate

### Onions OS Version

4.3.1-1

### Miyoo Firmware Version

2023 06 28 2128

### Provide a clear and concise description of the issue

![image](https://github.com/user-attachments/assets/a8c13a28-77f4-4e4a-a924-5609b3b38325)
![image](https://github.com/user-attachments/assets/87045c8a-15d3-4902-886c-761add234438)
![image](https://github.com/user-attachments/assets/1b3b7221-82c4-49ed-a2f6-5f5acb574ec9)
![image](https://github.com/user-attachments/assets/5da2e1ea-bfcf-400f-ac6a-e2e1dec184f9)

The PSX game in question does not work. Neither do the MediEvil ones and many others. OnionOS was freshly installed using the OnionOS Desktop Tools software for Windows. No issues occurred during installation. Only the Verified PSX method was selected when continuing the installation on the Miyoo Mini Plus. Using an official MicroSD card.

### Steps to Reproduce

- Perform a fresh OnionOS install using the OnionOS Desktop Tools for Windows
- Select only the Verified method for PSX and touch no other option
- Insert back the MicroSD card into your PC to transfer the BIOS and the ROMs
- Perform a Refresh All ROMs at the Games tab on the Miyoo Mini Plus
- PSX games don't work 

### Provide any additional context or information that may help us investigate your issue

v4.3.0 had no such issues in the past and v4.3.1-1 was installed using the formatting option in the OnionOS Desktop Tools for Windows. Everything was wiped by the tool before the latest OnionOS was installed.

**Comment by hotgar** [comment]
@mikhaelcool7 
You could get additional info by capturing logs on your device.

- Turn on the  `Apps->Tweaks->Advanced->Diagnostics->Enable logging` option
- Try launching the game that you have problems with
- Launch `Apps->Tweaks->Advanced->Diagnostics->Util: System log snapshot`
- Wait until top left of your screen says `Script successfully completed.`
- Turn `Enable logging` off

Doing the steps above should create a file `log_export.7z` in the root of your sd card. It should contain a number of logs including a retroarch log. You can inspect those logs to try and troubleshoot by yourself or you could upload the archive here.
If you're uncomfortable with sharing file listings of your sd card then uploading only the `logs` and `sysinfo` subdirectories of that archive should be sufficient.

**Reply by mikhaelcool7** [reply]
[log_export.zip](https://github.com/user-attachments/files/16488588/log_export.zip)
The tool to generate CUE files for BIN also fails

**Reply by hotgar** [reply]
**Edit:** Actually it looks like your .cue file points to `Colin McRae Rally 2.0 (USA) (En,Fr,Es) (Track 1).bin` but your Roms directory only has `Colin McRae Rally 2.0 (Track 1).bin` and the same thing with the `(Track 2)` file. Have you renamed the files by any chance?
